src/main/java/duke/ui/Ui.java

```diff
@@ -193,6 +220,30 @@ public void exportMessage() {
         printDash();
     }
 
+    /**
+     * This function tells the user that SpongeBob has exported the CSV file.
+     */
+    public void exportSelect() {
+        printSpaces("Noted. I have exported the selected details to a new CSV file");
+        printDash();
+    }
+
+
+    /**
```

Comment:
remove functions keywordPrint and remindersList

Reply:
removed